backend/inspirehep/records/api/base.py

```diff
@@ -230,6 +230,9 @@ def get_linked_records_from_dict_field(cls, data, path):
             return cls.get_records_by_pids(pids)
         return iter([])
 
+    def copy(self):
```

Comment:
I would add a docstring explaining why this is needed:
```suggestion
    def copy(self):
    """Copy the record metadata.
    
    This is needed because the default ``dict.copy`` always returns a ``dict``.
    """
```